.github/workflows/r-ci.yml

```diff
@@ -40,11 +40,11 @@ jobs:
       - name: Install BioConductor package SingleCellExperiment
         run: cd apis/r && tools/r-ci.sh install_bioc SingleCellExperiment
 
-      - name: Install rc version of tiledb-r (macOS)
+      - name: Install r-universe build of tiledb-r (macOS)"
```

Comment:
stray quote marks

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/1681
Thank you @thetorpedodog ! :)